### DISCUSSION TITLE
I am having troubles to set up chat on AWS hosted centos env - CORS?

### DISCUSSION THREAD

**Discussion by macin** [mckaywrigley/chatbot-ui#1295] [Help]
On the first connection to DW I see this... any ideas how to solve the CORS issue without having to migrate the website to HTTPS? I need to keep it http for now...

![image](https://github.com/mckaywrigley/chatbot-ui/assets/1422747/715e7c39-7c22-4790-a712-75ac8661a7f5)

**Comment by abhayk05** [comment]
I am having the same issue. Were you able to solve this ?
